plugin/plugin.go

```diff
@@ -23,35 +23,75 @@ const (
 
 	// CommandDescribeKey is the name of the plugin command
 	// which must be supported by every plugin that has the
-	// SIGNATURE_GENERATOR capability.
+	// SIGNATURE_GENERATOR.RAW capability.
 	CommandDescribeKey Command = "describe-key"
 
 	// CommandGenerateSignature is the name of the plugin command
 	// which must be supported by every plugin that has the
-	// SIGNATURE_GENERATOR capability.
+	// SIGNATURE_GENERATOR.RAW capability.
 	CommandGenerateSignature Command = "generate-signature"
 
 	// CommandGenerateEnvelope is the name of the plugin command
 	// which must be supported by every plugin that has the
-	// SIGNATURE_ENVELOPE_GENERATOR capability.
+	// SIGNATURE_GENERATOR.ENVELOPE capability.
 	CommandGenerateEnvelope Command = "generate-envelope"
+
+	// CommandVerifySignature is the name of the plugin command
+	// which must be supported by every plugin that has
+	// any SIGNATURE_VERIFIER.* capability
+	CommandVerifySignature Command = "verify-signature"
 )
 
 // Capability is a feature available in the plugin contract.
 type Capability string
 
 const (
-	// CapabilitySignatureGenerator is the name of the capability
-	// which should support a plugin to support generating signatures.
-	CapabilitySignatureGenerator Capability = "SIGNATURE_GENERATOR"
+	// CapabilityRawGenerator is the name of the capability
+	// for a plugin to support generating raw signatures.
+	CapabilityRawGenerator Capability = "SIGNATURE_GENERATOR.RAW"
```

Comment:
IMO, we can retain the original name, `CapabilityRawGenerator` misses the context that it's generating a signature.

```suggestion
	CapabilitySignatureGenerator Capability = "SIGNATURE_GENERATOR.RAW"
```